integration/tests/base.py

```diff
@@ -28,7 +28,7 @@ def prepare(self, operator: Operator, faucet: Faucet, web3_client, sol_client):
     def prepare_account(self, prepare_account):
         self.acc = prepare_account
 
-    def create_tx_object(self, sender, recipient=None, amount=0, nonce=None, gas_price=None, data=None,
+    def create_tx_object(self, sender, recipient=None, value=None, amount=0, nonce=None, gas=None, gas_price=None, data=None,
```

Comment:
value and amount look the same. Are you sure that we need both?

Reply:
we use "value" key in transaction. it seems reasonable to have arg value. You can use amount when it is transfer. It is a bit confusing for me to use arg "amount" for these transactions that are nothing common with transfers at all

Reply:
@gigimon what do you think

Reply:
I agree with @kristinaNikolaevaa it's one parameter with one difference, transform to ether or not. And we should use only one. I added amount and transform it to Ether because it's more human readable than wei.

Will better if we will convert all to Ether or (worse) add parameter: amount_cur with paramaters: wei, gwei, ether and convert inside

@romanova-natasha I see that you pass as value result from receipt, please convert it to ether and sent.

Reply:
@kristinaNikolaevaa I added a common method for creating tx, I call it inside create_tx_obj with value=amount

Reply:
but you forget remove value from create_tx_object

Reply:
But you leave both : value=None, amount=0 for this method
Sorry, I still dont understand why we need create_common_tx_obj and  create_tx_object 
there are no difference except value (why you cant use ethers as in another tests?)
but okey, it is not big issue of course